README.md

```diff
@@ -103,11 +103,11 @@ can be insalled via the system package manager.
 #### Compile
 
 ```sh
-cmake -S . -B build -G Ninja
-cmake --build build/
+cmake --preset linux-release
+cmake --build --preset linux-release
 
 # Optionally, if you want to install from source
-cmake --build build/ --target install
+cmake --build --preset linux-release --target install
 ```
```

Comment:
this also only addresses Linux OS (see below)

Reply:
I wil add info mac os, and for freebsd i suppose that one can use linux one as well